src/main/java/com/salesforce/dataloader/controller/Controller.java

```diff
@@ -376,15 +439,15 @@ protected void initConfig(String name, boolean isBatchMode) throws ControllerIni
     /**
      * @return product name
      */
-    private String getProductName() {
+    private static String getProductName() {
         return APP_NAME + " " + APP_VERSION;
     }
 
     public static synchronized void initLog() throws FactoryConfigurationError {
         // init the log if not initialized already
         if (Controller.isLogInitialized) { return; }
         try {
-            File logConfXml = new File(System.getProperty("user.dir"), LOG_CONF_OVERRIDE);
+            File logConfXml = new File(getDefaultConfigDir(), LOG_CONF_OVERRIDE);
```

Comment:
I thought about this and concluded that log setting might be something it should be set across for all users instead of each user controlling it. The log file is persisted in system-wide location.

I just realized that the log file is persisted in system-wide location in Mac but not in Windows.
Windows - `C:\Users\{user}\AppData\Local\Temp\`
Mac -  `/var/folders/…`

Do you think logs should be stored in user-specific or system-wide location?